src/language/parser.js

```diff
@@ -1512,21 +1508,31 @@ function expect(lexer: Lexer<*>, kind: TokenKindEnum): Token {
 }
 
 /**
- * If the next token is a keyword with the given value, return that token after
- * advancing the lexer. Otherwise, do not change the parser state and return
- * false.
+ * If the next token is a keyword with the given value, return true after advancing
+ * the lexer. Otherwise, do not change the parser state and return false.
  */
-function expectKeyword(lexer: Lexer<*>, value: string): Token {
+function skipKeyword(lexer: Lexer<*>, value: string): boolean {
```

Comment:
Since we already have skip/expect pair with matching functionality I think we need to have skipKeyword/expectKeyword.